### PR TITLE
#484, update pinned thread titles for newly posted threads when possible

### PR DESCRIPTION
**Pull request by Adamantcheese**
I have left an explanation as a comment.

### CODE REVIEW

**Comment by Adamantcheese** [comment]
At this point I'm not going to submit any other pull requests. I think that the 8 I put in are plenty more than enough. [If anyone's reading this, they can find a debug multi-release of all the features from the pulls here.](https://github.com/Adamantcheese/Clover/releases)

File: Clover/app/src/main/java/org/floens/chan/core/manager/WatchManager.java [hunk 1]
```diff
@@ -761,6 +761,15 @@ public void onChanLoaderError(ChanLoaderException error) {
...
             pin.isError = false;
+            /*
+             * Forcibly update watched thread titles
+             * This solves the issue of when you post a thread and the site doesn't have the thread listed yet,
+             * resulting in the thread title being something like /test/918324919 instead of a proper title
+             *
+             * The thread title will be updated as soon as the site has the thread listed in the thread directory
+             * 
+             */
+            pin.loadable.setTitle(PostHelper.getTitle(thread.op, pin.loadable));
```
Comment (andyklimczak) [inline Clover/app/src/main/java/org/floens/chan/core/manager/WatchManager.java:772]: not sure how expensive `.setTitle` is, but maybe we should `.setTitle` only if the current `pin.loadable.title` does not equal `PostHelper.getTitle(thread.op, pin.loadable)`. something like:
```
if (!pin.loadable.getTitle().equals(PostHelper.getTitle(thread.op, pin.loadable)) {
  pin.loadable.setTitle(PostHelper.getTitle(thread.op, pin.loadable));
}
```

minor thing really I think, so not needed

**Review by andyklimczak — APPROVED** [review]
LGTM

**Comment by Adamantcheese** [comment]
This might also be a solution to #450, but it might not.

**Pull request closed without merging**